scripts/defutil.py

```diff
@@ -152,18 +168,181 @@ def merge_pins(output, input_lef, def_one, def_two):
         f.write(merge_item_section("PINS", def_one_str, def_two_str))
 
 
-@click.command("replace_pins")
+# openroad -python scripts/defutil.py replace_pins -o output.def -l designs/def_test/runs/RUN_2022.01.30_12.32.26/tmp/merged.lef designs/def_test/runs/RUN_2022.01.30_12.32.26/tmp/floorplan/4-io.def designs/def_test/def_test.def
+
+
+@click.command(
```

Comment:
`click` idiomatically uses a docstring to print help for things.

Reply:
Will fix.